cms/djangoapps/contentstore/tests/test_proctoring.py

```diff
@@ -74,6 +75,10 @@ def test_publishing_exam(self, is_time_limited, default_time_limit_minutes,
         a proctored exam, it will also put an entry into the exam tables
         """
 
+        hide_after_due = False
+        if not is_proctored_exam and not is_practice_exam:
```

Comment:
All 3 kinds of special exams (timed, proctored, practice) will have a `hide_after_due` field, which defaults to `False`. So we're not really passing a value here so much as saying "publish an exam with the defaults".

Since this test is specifically checking `Happy path testing to see that when a course is published which contains a proctored exam, it will also put an entry into the exam tables`, I'm going to rip out the specialized logic here and just use the default value of false for all exam types.

There is/will be coverage for the functionality of `hide_after_due` including both possible values, but this test isn't the right place for it.